docs/unity_specific.md

```diff
@@ -11,7 +11,7 @@ This document lists some Unity related oddities and how we support EOS SDK in ou
 ----------------------------------------------------------------------------------------
```

Comment:
Top header should probably be something like "Unity-specific plugin functionality"